docs/device_support_guide.md

```diff
@@ -0,0 +1,108 @@
+# Intro
+To get your MSI laptop supported, there are two main methods, one that
+requires windows to be installed, and the other works directly on linux.
+
+If you have any bios/firmware updates on the official MSI website, and
+you haven't updated yet then **DO NOT UPDATE!** follow the guide first:
+
+## Windows method(recommended):
+
+1. Install Windows 10/11 normally, booting directly from a live usb or
+any other trick won't work, however windows activation is not
+needed.
+
+2. After a successful installation, make sure to download/install all
+Windows updates from the settings
+(windows can't update bios/firmware automatically on MSI
+laptops but that doesn't mean you have to update them yourself,
+instead continue following the guide).
+
+3. Install the MSI app designed for your laptop: MSI dragon center /
+MSI creator center / MSI center / MSI center pro (the correct app
+can be found on the support page for your laptop).
+
+4. If you fail at step 3, then you most likely need another app. usually
+its AMD Adrenalin software / GeForce Experience (nvidia)/ or intel
+equivalent.
+
+5. Once the MSI app is installed, you can test the functions it offers,
+like user scenario and battery charge limit.
+
+6. If everything works as expected, download RWEverything:
+https://rweverything.com/download/ ![download RWEverything](pics/support_guide/dl_rwe.png)
+
+7.  Launch it as administrator:
+![run as admin](pics/support_guide/run_as_admin.png)
+
+8. Navigate to the EC tab (page):
+![open ec tab](pics/support_guide/open_ec_tab.png)
+
+9. Here you should see a table of all the values your Embedded Chip has in its memory.\
+The values you see can be changed manually (by writing to them)\
+\
+DO NOT DO THAT: writing the wrong the values to the wrong address might brick the laptop completely and EC/BIOS RESET CAN'T FIX THAT!<br/>
```

Comment:
maybe being straightforward is not the best way to keep people from messing up, i prefer it my way because it sounds scarier. but im not against the change.

Reply:
Mention of the ability to write the data into the EC should definitely be fused together with the warning, otherwise the first paragraph by itself reads "here's the data and you can edit it".

I think making the *whole* warning bold and the damage&brick part ALL CAPS would be scary enough (I edited the suggestion)

Reply:
Github have this and similar tags, so ve can use them to highlight some dangerous things

> [!IMPORTANT]
qq

Reply:
Caution looks like the right one, let's see

> [!CAUTION]
**Do not attempt to edit the memory manually**. Writing the wrong the values to the wrong addresses will **DAMAGE YOUR HARDWARE** or even **BRICK YOUR DEVICE**! This can't be reversed by resetting the EC or BIOS.

Reply:
i just refreshed the page and the first thing that caught my eyes are the words in bold, lets keep the caution section because it looks good, but its not as eye catching as BIG **BOLD** words

Reply:
Nice.